### ISSUE TITLE
Different orientations between predicted segmentations and ground truth?

### ISSUE THREAD

**Issue by huilinyang** [fkong7/MeshDeformNet#13]
Hi, I tried to evaluate the predicted segmentations (from the predicted meshes) use the official evaluation code provided by MMWHS dataset. However, I found there are two CTs and almost all MRs have different orientations from the predicted. Cause I am a new cookie on medical images. Could you please give me some guides how to align them?

Thanks a lot!!!

**Comment by Thomas-Pr** [comment]
Hello, I have encountered the same issue when running your prediction script on the original MM-WHS images. I ran the data augmentation, preprocessing and training scripts exactly like you indicated in [your README](https://github.com/fkong7/MeshDeformNet/blob/main/README.md), using the same parameters that you provided and without modifying the code (except in the Laplacian loss in training to fix the error mentioned in issue #5, according to [your advice](https://github.com/fkong7/MeshDeformNet/issues/5#issuecomment-1291191142)) . However, in the end there seems to be an issue with the orientation of predicted meshes, which results in truncated segmentations, and in turn the average Dice value is 0 or close. Were there some changes needed to fix this that maybe did not make it to the final repository? Your help will be greatly appreciated!
